Rakefile

```diff
@@ -94,6 +96,11 @@ DocTest::RakeTasks.new do |t|
   t.converter_opts = { backend_name: 'revealjs' }
 end
 
+Rake::TestTask.new(:test) do |t|
```

Comment:
I've created a test task to run tests in the `test/asciidoctor-revealjs` directory.
Previously, the existing file `test/asciidoctor-revealjs/version_test.rb` was not executed.